### PR TITLE
Automatically configure version information after every chagelog generation

### LINKED ISSUE #481: automatically configure version information
As far as I understand new releases trigger integrating release notes. That's cool, but what I still do manually is [updating this file](https://github.com/sagemath/website/blob/master/conf/config.yaml) (2x version and 1x ISO date)

This could be automated as well (reading in the yaml, fixing these 3 fields, then saving it)

/cc @soham30rane @kwankyu

### PR DESCRIPTION
**Pull request by soham30rane**
Fixes #481

This will allow the `generate_changelog.yml` to also do the following
 - Fetch latest stable release (this is done explicitly because the currently generated changelog might be for different release than the latest one)
 - Update version and release date in `conf/config.yaml`

### CODE REVIEW

File: scripts/create_changelog.py [hunk 1]
```diff
@@ -407,6 +407,41 @@ def save_to_file(filename, ver, date_of_release):
...
+        release_date (str): Release date to set
+        config_file_path (str): Path to the config file
+    """
+    with open(config_file_path, 'r') as file:
+        content = file.read()
+
+    content = re.sub(r'releasedate:\s*".*?"', f'releasedate: "{release_date}"', content)
+    content = re.sub(r'version:\s*".*?"', f'version: "{ver}"', content)
+    content = re.sub(r'version_src:\s*".*?"',f'version_src: "{ver}"',content)
+
```
Comment (kwankyu) [inline scripts/create_changelog.py:442]: In https://github.com/sagemath/website/commit/fae9b541aa1fe2c3ade7800ca5d3f6dee6366204, I changed `releasedate` to `release_date`.

Comment (kwankyu) [reply scripts/create_changelog.py:442]: In line 441, `, content`.

File: scripts/create_changelog.py [hunk 2]
```diff
@@ -450,6 +485,9 @@ def save_to_file(filename, ver, date_of_release):
     all_contribs = sorted(all_contribs, key=lambda x: (x[0].startswith('@'), x[0]))
     if all_info:
         save_to_file(filepath, ver, date_of_release)
+        latest_release = get_latest_stable_release()
+        if latest_release:
+            update_config(latest_release['tag'],latest_release['date'],'conf/config.yaml')
     else:
```
Comment (kwankyu) [inline scripts/create_changelog.py:491]: `, 'conf/config.yaml')`

**Comment by kwankyu** [comment]
@haraldschilly In https://github.com/sagemath/website/commit/fae9b541aa1fe2c3ade7800ca5d3f6dee6366204, I changed `releasedate` to `release_date`. Hopefully, this change does not affect your workflow.

**Comment by soham30rane** [comment]
I ran autopep8 on the entire file to ensure good formatting. This may result in unrelated formatting changes showing up in the diff, but no functional changes were made beyond the intended scope of the PR

**Review by kwankyu — APPROVED** [review]
LGTM.

**Comment by kwankyu** [comment]
Thanks!

**Comment by haraldschilly** [comment]
> @haraldschilly In [fae9b54](https://github.com/sagemath/website/commit/fae9b541aa1fe2c3ade7800ca5d3f6dee6366204), I changed `releasedate` to `release_date`. Hopefully, this change does not affect your workflow.

Thx for the heads up, but as long as you also fix the template variables, everything is fine :rainbow:

**Pull request merged**